typescript/sdk/src/core/EvmCoreModule.hardhat-test.ts

```diff
@@ -161,4 +167,34 @@ describe('EvmCoreModule', async () => {
       expect(await timelockControllerContract.getMinDelay()).to.equal(DELAY);
     });
   });
+
+  describe('Update', async () => {
+    it('should update the owner only if they are different', async () => {
+      const evmCoreModule = await EvmCoreModule.create({
+        chain: CHAIN,
+        config,
+        multiProvider,
+      });
+      const newOwner = randomAddress();
+      let latestConfig = normalizeConfig(await evmCoreModule.read());
+      expect(latestConfig.owner).to.not.equal(newOwner);
+
+      await sendTxs(
+        await evmCoreModule.update({
+          ...config,
+          owner: newOwner,
+        }),
+      );
```

Comment:
please set to var before passing in

```suggestion
      const updateTxs = await evmCoreModule.update({ // should still change naming convention imo, i.e. why would "update" return unexecuted transactions
          ...config,
          owner: newOwner,
      });

      await sendTxs(updateTxs);
```

Reply:
actually a nit, forget this was a test file :)